### ISSUE TITLE
Builtin-Users API doc not clear how to specify user with JSON

### ISSUE THREAD

**Issue by tomck** [IQSS/dataverse#3192]
https://github.com/IQSS/dataverse/blob/develop/doc/sphinx-guides/source/api/native-api.rst

> Generates a new user. Data about the user are posted via JSON. Note that the password is passed as a parameter in the query.

This information isn't very clear, creating issue per @pdurbin in IRC

**Comment by pdurbin** [comment]
@tomck thanks for creating this issue. http://guides.dataverse.org/en/4.4/installation/installation-main.html#superuser-account mentions `dvinstall/data/user-admin.json` and as of this writing it looks like this:

```
$ cat dvinstall/data/user-admin.json 
{
    "firstName":"Dataverse",
    "lastName":"Admin",
    "userName":"dataverseAdmin",
    "affiliation":"Dataverse.org",
    "position":"Admin",
    "email":"dataverse@mailinator.com"
}
```

This comes from https://github.com/IQSS/dataverse/blob/v4.4/scripts/api/data/user-admin.json and there are similar files in that directory.

I have some crazy scripts I use in testing here if they are of interest:

- https://github.com/IQSS/dataverse/blob/v4.4/scripts/search/populate-users
- https://github.com/IQSS/dataverse/blob/v4.4/scripts/search/create-users

They operate on https://github.com/IQSS/dataverse/blob/v4.4/scripts/search/users.tsv

Probably too much information. 😄 

I agree the API Guide should be made more clear.

**Comment by dlmurphy** [comment]
Worked with @pameyer on this during the hackathon. We added a JSON example to help make this more clear.

**Comment by pdurbin** [comment]
@dlmurphy @pameyer in your pull request, don't forget to use the magic "connects to" syntax and advance the issue to code review in Waffle.

**Comment by pameyer** [comment]
Connects to #3192

**Comment by pdurbin** [comment]
@pameyer I could be wrong but I think the "connect to" needs to go in the description.

**Comment by pdurbin** [comment]
I tried to add some clarify in bffb3fe and moved this issue to Code Review at https://waffle.io/IQSS/dataverse

**Comment by pdurbin** [comment]
This was merged as part of pull request #3892.

**Comment by pdurbin** [comment]
Whoops, I meant to say that the fix is in pull request #3920.
